vendor/ansible_collections/nvidia/bare_metal/plugins/module_utils/client.py

```
@@ -0,0 +1,214 @@
+
+            url = self._url(full_path)
+            try:
+                resp = open_url(
+                    url,
+                    method='GET',
+                    headers=self._headers(),
+                    timeout=30,
+                )
+                status = resp.getcode()
+                raw = resp.read()
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)